config/mailjet.php

```diff
@@ -2,12 +2,12 @@
 
 declare(strict_types=1);
 
-
 use App\Mails\Template\TemplateName;
 
 return [
     'enabled' => env('MAILJET_TRANSACTIONAL_EMAILS_ENABLED', false),
     'sandbox_mode' => env('MAILJET_SANDBOX_MODE', true),
+    'sandbox_allowed_domains' => env('MAILJET_SANDBOX_ALLOWED_DOMAINS', []),
```

Comment:
This can now return two different types. When `MAILJET_SANDBOX_ALLOWED_DOMAINS` is provided it will return a comma-separated string, when it is not provided it will return an empty array. To avoid this I would put the conversion from string to array here.